demos/common/python/models/yolo.py

```diff
@@ -211,3 +211,75 @@ def postprocess(self, outputs, meta):
             detections = self._resize_detections(detections, meta['original_shape'][1::-1])
 
         return detections
+
+
+class YoloV4(YOLO):
+    class Params:
+        def __init__(self, sides, mask):
+            self.num = 3
+            self.coords = 4
+            self.classes = 80
+            self.sides = sides
+            self.anchors = [12.0, 16.0, 19.0, 36.0, 40.0, 28.0,
```

Comment:
```suggestion
            anchors = [12.0, 16.0, 19.0, 36.0, 40.0, 28.0,
```

`self.anchors` is unconditionally reassigned later, so there's no reason to assign it here.